namespaces/aushape.yml

```diff
@@ -0,0 +1,55 @@
+
+namespace:
+  name: aushape
+  type: group
+  description: >
+    Audit events converted with aushape
+    For more information please see
+    https://github.com/Scribery/aushape
+  fields:
+  - name: serial
+    type: long
+    description: >
+      Audit event serial number
+  - name: time
+    type: date
+    description: >
+      Audit event timestamp
```

Comment:
Theoretically, this is the time the audit event happened. It will probably also be local to the host that generated that event, which could be delivered using audisp-remote to this machine. I'm not sure if it is useful to keep a separate timestamp, perhaps we should use this as the `@timestamp` field.